docs/dev_guide/design_appendix.md

```diff
@@ -0,0 +1,76 @@
+---
```

Comment:
I suggest removing this file. It is too volatile in nature in some of this is already referenced in DESIGN.md